orchagent/aclorch.cpp

```diff
@@ -1168,6 +1200,13 @@ bool AclTable::create()
         table_attrs.push_back(attr);
     }
 
+    if (type == ACL_TABLE_L3)
```

Comment:
why only  support this on L3 table?

Reply:
This rule may be supported on all ACL tables, but OUT_PORTS may be not supported by all vendors, so only support this on L3 table currently. ACL_TABLE_L3 can be removed later.